### DISCUSSION TITLE
Is it possible to make GeneralErrorBoundary more powerful?

### DISCUSSION THREAD

**Discussion by nichtsam** [epicweb-dev/epic-stack#469] [General]
I was trying to integrate my side-project with `epic-stack`'s error-boundary pattern.
I added `const location = useLocation();` into `GeneralErrorBoundary`,\
and I'm passing down `error`, `params`, and `location` to both `statusHandlers` and `unexpectedErrorHandler`.

I wonder if there is any downside to this?

**Comment by kentcdodds** [comment]
Nope, that's fine :)

Can you tell me why you needed the `location`?

**Reply by nichtsam** [reply]
I find this information valuable for both route error handling and enhancing the user experience.
